spec/requests/races_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RACES_NUMBER = 5
```

Comment:
don't use constants in test. If you need something to be set for all you're test cases you can use`let`